openedx/core/djangoapps/oauth_dispatch/tests/test_views.py

```diff
@@ -231,6 +231,36 @@ def test_jwt_access_token(self, client_attr):
             should_be_restricted=False,
         )
 
+    @ddt.data('dop_app', 'dot_app')
+    def test_jwt_access_token_from_header(self, client_attr):
+        client = getattr(self, client_attr)
+        response = self._post_request(self.user, client, HTTP_X_TOKEN_TYPE='jwt')
```

Comment:
Take a look at [the implementation](https://github.com/edx/edx-platform/blob/a4f03620a41b4d8b6a0de1f3bc5581e505230b17/openedx/core/djangoapps/oauth_dispatch/tests/test_views.py#L126) of `_post_request`.  You'll need to update it to accept the new header parameter so it can pass `HTTP_X_TOKEN_TYPE` through to the `self.client.post` call.